src/main/java/com/lb/domain/AccommodationEvent.java

```diff
@@ -0,0 +1,8 @@
+package com.lb.domain;
+
+public sealed interface AccommodationEvent {
+
+  record AccommodationFound(Accommodation accommodation) implements AccommodationEvent {}
```

Comment:
these should have `@TypeName` annotations

Reply:
ofc. Added